<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Booking</class>
 <widget class="QDialog" name="Booking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1101</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubmitDelete">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Deleted</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubmitAdd">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Added</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Count">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Count Price</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
